packages/visx-demo/src/sandboxes/visx-brush/Example.tsx

```diff
@@ -113,6 +115,38 @@ function BrushChart({
     [brushDateScale],
   );
 
+  const handleResetClick = () => {
+    if (brushRef?.current) {
+      setFilteredStock(stock);
+      brushRef.current.reset();
+    } else {
+      console.log(`innerRef Prop not properly set.`);
+    }
+  };
+
+  const handleLastWeekClick = () => {
+    if (brushRef?.current === null) {
+      console.log(`innerRef Prop not properly set.`);
```

Comment:
the console logs are useful in debugging, but for the simplicity of the demo (which should work!) I wonder if we could remove them?